pytest/tests/replay/replay.py

```diff
@@ -0,0 +1,161 @@
+        json.dump(key_json, fout, indent=2)
+
+def prompt_to_launch_localnet():
+    input('Please launch your localnet node now and press enter to continue...')
+
+def convert_snack_case_to_camel_case(s):
+    words = s.split('_')
+    return words[0] + ''.join(w.title() for w in words[1:])
+
+def convert_dumped_rust_instance_to_py_object(ordered_dict_instance, class_name):
+    ans = class_name()
+    for attr_key, attr_value in ordered_dict_instance.items():
+        setattr(ans, convert_snack_case_to_camel_case(attr_key), attr_value)
+    return ans
+
+def convert_transaction_type_string_to_class(name):
+    if name == "CreateAccount":
+        return CreateAccount
+    elif name == "DeleteAccount":
+        return DeleteAccount
+    elif name == "DeployContract":
+        return DeployContract
+    elif name == "FunctionCall":
+        return FunctionCall
+    elif name == "Transfer":
+        return Transfer
+    elif name == "Stake":
+        return Stake
+    elif name == "AddKey":
+        return AddKey
+    elif name == "DeleteKey":
+        return DeleteKey
+    raise ValueError('Unknown tx type: %s' % name)
+
+def convert_dumped_public_key_to_py_public_key(dumped_public_key):
+    pk_str = dumped_public_key.split(':')[1] if ':' in dumped_public_key else dumped_public_key
+    ans = PublicKey()
+    ans.keyType = 0
+    ans.data = base58.b58decode(pk_str.encode('ascii'))
+    return ans
+
+def fix_dumped_fields_by_tx_type(py_tx, tx_type):
+    if tx_type == "CreateAccount":
+        pass
+    elif tx_type == "DeleteAccount":
+        pass
+    elif tx_type == "DeployContract":
+        py_tx.code = base64.b64decode(py_tx.code.encode('ascii'))
+    elif tx_type == "FunctionCall":
+        py_tx.args = base64.b64decode(py_tx.args.encode('ascii'))
+        py_tx.deposit = int(py_tx.deposit)
+    elif tx_type == "Transfer":
+        py_tx.deposit = int(py_tx.deposit)
+    elif tx_type == "Stake":
+        py_tx.stake = int(py_tx.stake)
+        py_tx.publicKey = convert_dumped_public_key_to_py_public_key(py_tx.publicKey)
+    elif tx_type == "AddKey":
+        raise ValueError('Unsupported tx type: %s' % tx_type)
+    elif tx_type == "DeleteKey":
+        py_tx.publicKey = convert_dumped_public_key_to_py_public_key(py_tx.publicKey)
+    else:
+        raise ValueError('Unknown tx type: %s' % tx_type)
+    return py_tx
+
+def convert_dumped_action_to_py_action(action_dict):
+    assert(len(action_dict) == 1)
+    for tx_type, value in action_dict.items():
+        contents = convert_dumped_rust_instance_to_py_object(value, convert_transaction_type_string_to_class(tx_type))
+        contents = fix_dumped_fields_by_tx_type(contents, tx_type)
+        action = Action()
+        action.enum = tx_type[0].lower() + tx_type[1:]
+        setattr(action, action.enum, contents)
+        return action
+
+def send_resigned_transactions(tx_path, key_pair):
+    LOCALHOST = '127.0.0.1'
```

Comment:
this is a local variable -- use small letters (or if this is a const -- consider moving to top of the file)